spec/validators/closed_case_validator_spec.rb

```diff
@@ -74,7 +74,7 @@
       end
 
       it "does not error with blank date_responded when case is rejected" do
```

Comment:
Should this say "when case is invalid_submission"?

Reply:
I think this is OK, because it's saying the case has been rejected and the initial state of a rejected case is "invalid_submission".